main_test.py

```diff
@@ -1,11 +1,12 @@
 import sys
-sys.path.append('src')
 
+sys.path.append("src")
+
+
+# Are these supposed to be tests? tf
 import pychart
-import pychart.scanner
-import pychart.token_type
-import pychart.token_type.token_type_enum
-import pychart.token_type.token
```

Comment:
I think these should all be private and hidden from the user downloading the pip package... This means that the user can only import a few methods which give high level control for running the language and are no able to import internal stuff like tokens or the raw token scanner

Reply:
@spralja wanna merge this into your package PR? Might make it easier since we can close this pr and keep the changes in one place